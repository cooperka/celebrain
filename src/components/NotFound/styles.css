.NotFound {
  margin: 4em 0;
  font-weight: bold;
}

.NotFound .subtitle {
  margin: 1rem auto;
  font-size: 2.2em;
  max-width: 20em;
}
