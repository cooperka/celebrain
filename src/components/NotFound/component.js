// @flow

import React from 'react';
import { Link } from 'react-router-dom';

import './styles.css';

type Props = {
  location: any,
};

function NotFound(props: Props) {
  const { location } = props;

  return (
    <div className="NotFound">
      <div className="subtitle">
        The page at <code>{location.pathname}</code> doesn{'\''}t exist.
      </div>
      <div className="subtitle">
        Go back <Link to="/">home</Link>?
      </div>
    </div>
  );
}

export default NotFound;
