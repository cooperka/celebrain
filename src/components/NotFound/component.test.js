// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './component';

it('renders', () => {
  const tree = renderer.create(<NotFound location={{ pathname: '/test' }} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
