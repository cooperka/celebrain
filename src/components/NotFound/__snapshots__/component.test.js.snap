// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="NotFound"
>
  <div
    className="subtitle"
  >
    The page at 
    <code>
      /test
    </code>
     doesn
    '
    t exist.
  </div>
  <div
    className="subtitle"
  >
    Go back 
    <Link
      to="/"
    >
      home
    </Link>
    ?
  </div>
</div>
`;
